### ISSUE TITLE
False positives with rule `no-static-element-interactions` and drag and drop handlers?

### ISSUE THREAD

**Issue by Evertvdw** [vue-a11y/eslint-plugin-vuejs-accessibility#980]
Hello there! I just started using this ESLint extension, and I noticed it is flagging a lot of stuff of which I doubt is valid. 

I have HTML like:
```html
<div
  @dragover="dragOver"
  @dragleave="dragLeave"
  @dragenter="preventDefault"
  @drop="onDrop"
>
```

Which will trigger the rule `no-static-element-interactions` but is that correct? I cannot put a role attribute on this element because there is no role for an element being a valid dropping target. I can of course just ignore that rule in this case but I'm raising the issue here because maybe some event listeners should be excluded from triggering this rule, like the 4 above here.

What do you think?

**Comment by NathanBnm** [comment]
I have the same issue with the following:

```vue
<li
   draggable
   @dragstart="onDragStart"
   @dragend="onDragEnd"
</li>
```

It should not trigger the error because `<li>` can be moved using drag and drop inside a list
